src/main.rs

```diff
@@ -162,14 +176,41 @@ async fn main() {
     let default_keypair = args.keypair.unwrap_or(cli_config.keypair_path.clone());
     let fee_payer_filepath = args.fee_payer.unwrap_or(default_keypair.clone());
     let rpc_client = RpcClient::new_with_commitment(cluster, CommitmentConfig::confirmed());
-
+    let jito_client =
+        RpcClient::new("https://mainnet.block-engine.jito.wtf/api/v1/transactions".to_string());
```

Comment:
Please add a `--jito-url` argument so we can set preferred region as per https://jito-labs.gitbook.io/mev/searcher-resources/json-rpc-api-reference/url#urls